optaplanner-docs/src/main/asciidoc/DroolsScoreCalculation/DroolsScoreCalculation-chapter.adoc

```diff
@@ -0,0 +1,297 @@
+[[droolsScoreCalculation]]
+= Drools Score Calculation
+:doctype: book
+:imagesdir: ..
+:sectnums:
+:toc: left
+:icons: font
+:experimental:
+
+
+[[droolsScoreCalculationOverview]]
+== Overview
+
+Implement your score calculation using the Drools rule engine.
+Every score constraint is written as one or more score rules.
+
+* Advantages:
```

Comment:
Good point, but for now, I just moved this SWOT documentation. I intend to revisit it when constraints streams are implemented, at which point it will be a better timing to refactor this. We might just move the entire swot analysis into the score calculation chapter, so they are next to each other.